exercises/robot-simulator/robot_simulator_test.go

```diff
@@ -9,7 +9,7 @@ package robot
 //
 //    go test                      # run all tests
 //    go test -tags step1          # run just step 1 tests.
-//    go test -tags 'step1 step2'  # run step1 and step2 tests
+//    go test -tags step1,step2    # run step1 and step2 tests
```

Comment:
This isn't semantically same (correct in the updated version).

Reply:
I'll do a separate PR, I didn't notice I was in the master when pushing a second change.